markdown/manual.md

```diff
@@ -1,220 +1,245 @@
+### Performance Assessments
+Our organization is committed to a fair, collaborative, and straightforward review of each employee's performance.  We are in the process of determining the best methodology to carry out this commitment and will update this manual when a determination is made.
 
+### Raises and Bonuses
 Annual cost of living increases will be combined with any merit increases described above.  These will be reviewed and approved by the Board of Directors on an annual basis.
 
 Additional raises and bonuses may be considered at other times to account for special circumstances, such as extraordinary performance, a change in position and/or an increase in responsibilities.
 
+## Reimbursable Expenses
 
-## Covered Expenses & Reimbursements
-
-### Covered Expenses
-The organization will reimburse pre-approved, work-related expenses.  Any expenditure totaling more than $1,000—including cumulative expenses like software subscriptions—must be pre-approved by the Board of Directors and Executive Director.  Sub-$1,000 reimbursable expenses which do not require BOD approval include but are not limited to:
+### What's Covered
+The organization will reimburse pre-approved, work-related expenses.  Any expenditure totaling more than $1,000—including cumulative expenses like software subscriptions—must be pre-approved by the Board of Directors and Executive Director.  Sub-$1,000 reimbursable expenses which do not require Board approval include but are not limited to:
 
 * Travel between work meetings and the office.
 * Travel to, from, and during conferences and events.
 * Housing and food during conferences and events.
 * Business-meeting meals and drinks.
 * Office supplies.
 * Organization-specific subscriptions and/or software.
-* Expenses stemming from events organized by the organization
+* Expenses associated with organization sponsored events.
 
-Employees are expected to travel coach class when traveling by air, book reasonably-priced hotels and not spend excessively on meals and other travel expenses.
+Employees are expected to travel economy class when traveling by air, book reasonably-priced hotels and not spend excessively on meals and other travel expenses.
 
-### Reimbursements
-You are strongly encouraged to file reimbursement requests as soon as you incur the expenses, for example, within the week following a business trip or conference.  Expenses are usually reimbursed within a week of approval and submission.  Each reimbursement request must be sent to the Executive Director, who will review, approve, then submit for payment to the Chairman of the Board of Directors and our bookkeeper.  Each reimbursement request must include these three things:
+### How to File a Reimbursement Request
```

Comment:
I think we were exploring systems to automate this stuff, like Expensify, is that not gonna work out?

Reply:
@doshitan not *fully* gonna work out for the foreseeable future.  I use Expensify, but still have to manually wrangle to fit into our reporting and compliance formatting.  

@batemapf Can we add language about default to the ED buying stuff whenever possible?  Makes my life a sliver harder, but everyone's life way easier.